tests/compat/test_gecko44.py

```diff
@@ -0,0 +1,15 @@
+from helper import CompatTestCase
+from validator.compat import FX44_DEFINITION
+
+
+class TestFX44Compat(CompatTestCase):
+    """Test that compatibility tests for Gecko 44 are properly executed."""
+
+    VERSION = FX44_DEFINITION
+
+    def test_mozFetchAsStream(self):
+        self.run_script_for_compat(
+            'let stylesheets = window.getAllStyleSheets(config.browser.contentWindow);'
+        )
+        self.assert_silent()
+        self.assert_compat_error()
```

Comment:
There isn't a specific error id so we can't use that. It's mostly the fact that `run_script_for_compat` will only run the code against the fx44 version and only this code so the chance that there's another error is relatively thin. tldr; it doesn't, I can improve it a tiny bit though and check the message too.

We simply don't have that anywhere else though.